snowex_db/metadata.py

```diff
@@ -459,7 +462,9 @@ class ExtendedSnowExPrimaryVariables(SnowExPrimaryVariables):
         "ignore", "Ignore this",
         [
             "original_index", 'id', 'freq_mhz', 'camera',
-            'avgvelocity', 'equipment', 'version_number'
+            'avgvelocity', 'equipment', 'version_number',
```

Comment:
Also wondering if `equipment` or `version_number` should go into the `Instruments` table